### ISSUE TITLE
Debuffs block too much horizontal space from the healthbar

### ISSUE THREAD

**Issue by Feichtmeier** [GoldpawsStuff/AzeriteUI#85]
**What WoW version do you wish to see this feature in?**
- [ ] Classic
- [ X] Battle for Azeroth

**Is your feature request related to a problem? Please describe.**
When healing you need to watch over both debuffs and the health bar and then make a decision what to do. With the debuffs taking up 1/4 of horizontal space on the health bars you really can not see if you target is at 25% health (danger) or at 70% health (okay, wait some more GCDs). 

**Describe the solution you'd like**
It would be nice if the position of debuffs could be reconsidered, eventually above the avatars and a little bit smaller. I think placing them ontop of the health bar should be avoided.

**Describe alternatives you've considered**
Alternatively they could be made transparent so you see both the health bar and the debuff.

**Comment by goldpaw** [comment]
Not wrong. The same problem applies to both party- and raid frames, though, and in the raid frames, there really isn't anywhere to move them. 

Making them transparent is not an option, because... well, making them SO transparent that the health is clearly visible would make the debuffs invisible. Also, an element does not in its entirety become transparent, it's the separate layers that becomes transparent, meaning anything overlapping, like the borders, shadows, backdrops, all become visible at once. It's a mess. 

Totally agree that we need to see the actual health at all times, though. Leaving this open until I have a fix ready! :)

**Comment by goldpaw** [comment]
Yeah I did understand it. And that solution wouldn't work for raids. So I would need to come up with another solution regardless, thus not solving the issue.

**Comment by Feichtmeier** [comment]
![grafik](https://user-images.githubusercontent.com/15329494/111359134-6b144a00-868b-11eb-9fcf-9a73bcf4030b.png)
 Fast screenshot edit... I could think of two more options for raids and parties
Either put it ontop of the role symbol, but this could again lead into situations where you do not know who to dispell now (let the tank die or the dps warrior etc)
Or make it even smaller and put it at the top left of the health frame

**Comment by goldpaw** [comment]
Putting it outside the frame clutters up the space between frames, which is as important as the frames themselves. Shrinking it down to nothing negates the point of having it in the first place. It's supposed to be alarming. It's supposed to be "in your face". 

The problem isn't the debuff. It's the health.

**Comment by goldpaw** [comment]
...I can just move it dynamically horizontally to NOT cover the edge of the health? Keep it centered when that's no problem, move it to a side otherwise? Yup. I think that'll do it. I'm doing that! :)
